### PR TITLE
Port button styling into webpack

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Port button styling into webpack. Still gotta do a little additional tidying, but this means almost all our custom code is ported.

This pull request makes the following changes:
* Port button styling into webpack

screencaps posted inline

It relates to the following issue #s: 
* Bumps #1859 

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

File: app/javascript/styles/buttons.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+// btn-danger
+.btn-danger {
+  background-color: $dark-red !important;
+  border-color: $dark-red !important;
+}
+
+// btn-warning
+.btn-warning {
```
Comment (colinxfleming) [inline app/javascript/styles/buttons.scss:33]: Before: 
image

File: app/javascript/styles/buttons.scss [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  &:focus {
+    background-color: $daria-color;
+    &:hover {
+      background-color: $daria-color-lt;
+    }
+  }
+}
+
+// btn-success
+.btn-success {
```
Comment (colinxfleming) [inline app/javascript/styles/buttons.scss:54]: Before: (right side, the yes button) image

After: TK

File: app/javascript/styles/buttons.scss [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  &:focus {
+    background-color: $daria-color;
+    &:hover {
+      background-color: $daria-color-lt;
+    }
+  }
+}
+
+// btn sizes
+.btn-lg {
```
Comment (colinxfleming) [inline app/javascript/styles/buttons.scss:74]: Before: 
image

After: TK

File: app/javascript/styles/buttons.scss [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.btn-warning[disabled] {
+  @include disabled-button;
+}
+
+// Styling for the cancel button on pledge modal
+.btn[data-orientation='cancel'] {
+  color: $red;
+}
```
Comment (colinxfleming) [inline app/javascript/styles/buttons.scss:109]: Before: 
image

After: TK

**Review by colinxfleming — COMMENTED** [review]
notes

File: app/javascript/styles/buttons.scss (hunk 2 above)
Comment (colinxfleming) [reply app/javascript/styles/buttons.scss:54]: After: 
image

File: app/javascript/styles/buttons.scss (hunk 1 above)
Comment (colinxfleming) [reply app/javascript/styles/buttons.scss:33]: After: 
image

File: app/javascript/styles/buttons.scss [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+.btn-primary:focus {
+  background-color: lighten($daria-color,15%);
+  color: $white;
+  text-decoration: none;
+}
+
+// btn-danger
+.btn-danger {
```
[the remark this answers is not in the document]
Comment (colinxfleming) [reply app/javascript/styles/buttons.scss:27]: After: 
image

File: app/javascript/styles/buttons.scss [hunk 6]
```diff
@@ -0,0 +1,109 @@
+// Button styling.
+
+@import './bootstrap';
+@import './_variables';
+
+// btn-primary
+.btn-primary {
```
[the remark this answers is not in the document]
Comment (colinxfleming) [reply app/javascript/styles/buttons.scss:7]: After: 
image

File: app/javascript/styles/buttons.scss (hunk 3 above)
Comment (colinxfleming) [reply app/javascript/styles/buttons.scss:74]: After: image

(the spacing on this is a few pixels off but not substantially different)

File: app/javascript/styles/buttons.scss (hunk 4 above)
Comment (colinxfleming) [reply app/javascript/styles/buttons.scss:109]: After: 
image

File: app/assets/stylesheets/application.css.scss [hunk 7]
```diff
@@ -54,121 +54,3 @@ a:hover, a:active, a:visited, a:focus {
...
-
-
-.btn[data-orientation='cancel'] {
-  color: red;
-}
-.submit {
-  @extend .btn;
-  @extend .btn-primary;
-  @extend .btn-lg;
-  }
```
Comment (lomky) [inline app/assets/stylesheets/application.css.scss:174]: Did we mean to delete this one?

**Review by lomky — APPROVED** [review]
One q, but all looks good!

File: app/assets/stylesheets/application.css.scss (hunk 7 above)
Comment (colinxfleming) [reply app/assets/stylesheets/application.css.scss:174]: yep! This is an unused class, so we can remove the styling guilt free.

**Pull request merged**